doc/howto/ggplot-interact.md

```diff
@@ -193,8 +193,11 @@ show_legend = widgets.ToggleButton(
 
 ```{code-cell} ipython3
 def plot(b, cmap, show_legend):
-    (ggplot("diamonds", aes(x="price")) + geom_histogram(bins=b, fill="cut", cmap=cmap)
-     + facet_wrap("color", legend=show_legend))
+    (
```

Comment:
Please revert these formatting changes. It's not part of this PR. I see a lot of added newlines as well. 
Please run `pkgmt format` before pushing code

Reply:
I did pkgmt format and pkgmt lint before committing, need to look what is breaking